### DISCUSSION TITLE
problems with audio transcription

### DISCUSSION THREAD

**Discussion by benevenuti15** [sepinf-inc/IPED#1971] [General]
I'm trying to perform audio transcription indexing for the first time. Follow all procedures as instructed by Nassif in this thread. 

https://github.com/sepinf-inc/IPED/discussions/1822

However, when starting indexing, IPED remains stuck on the screen below without progressing with the process. In addition to the information provided in this thread, are there any other steps I should take?

![problems with audio transcription](https://github.com/sepinf-inc/IPED/assets/98331327/0caee1eb-b88f-4162-a449-73e775a8160c)

**Comment by wladimirleite** [comment]
Which transcription algorithm/service are you using?
How about the processing log? Any relevant messages? Can you share it?

**Reply by benevenuti15** [reply]
I used the procedures informed by Nassif in this topic: https://github.com/sepinf-inc/IPED/discussions/1822

I believe the algorithm is Facebook Wav2Vec2. I have attached an error log in Nassif's comment response below.

**Comment by lfcnassif** [comment]
I would also ask if you are using a proxy? If yes, it could be blocking the transcription model download... And was any message printed on the command line terminal?

**Reply by benevenuti15** [reply]
I work in a public agency, here there really is a proxy, but I ran the test on a network without a proxy, it still didn't work.


Attached is the error log displayed in the power shell terminal.

[IPED error log.txt](https://github.com/sepinf-inc/IPED/files/13311339/IPED.error.log.txt)

**Reply by lfcnassif** [reply]
Is this log from the machine with the proxy or without it?

From what I see into the log:
> ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1129)

Seems you are having a SSL Certificate validation issue. Maybe the proxy in your organization is making a MITM and replacing the certificate of the target site and your custom organization signer is not being validated by the trusted authority chain of your machine or IPED's python. This kind of MITM is common in organizations to inspect user encrypted traffic.

**Reply by lfcnassif** [reply]
If the problem is that explained above, installing the organization certificate into IPED's python certificate store may workaround the problem.
